user_model.ml

```diff
@@ -0,0 +1,66 @@
+module Rng = Mirage_crypto_rng
+
+type token = {
+  token_type : string;
+  access_token : string;
```

Comment:
can we rename this field to `id` or `value` or something similar?

Reply:
yes we can, value seems appropriate